python/pyspark/ml/classification.py

```diff
@@ -1172,6 +1175,53 @@ def getClassifier(self):
         """
         return self.getOrDefault(self.classifier)
 
+    @classmethod
```

Comment:
None of these additions to OneVsRestParams are needed to make the tests run.  I thought they were only needed for tuning.py persistence.  Are they used anywhere?